ZenPacks/zenoss/LinuxMonitor/HardDisk.py

```diff
@@ -34,4 +34,4 @@ def getPhysicalVolumes(self):
         return physicalvolumes
 
     def getRRDTemplateName(self):
-            return "HardDisk"
+            return "IOStat"
```

Comment:
The primary template bound to HardDisk should be called HardDisk, not IOStat. If you want to create an IOStat monitoring template for some reason, it should be in addition to HardDisk, not as a replacement for it.

I can't quite tell the reason for wanting to use an IOStat monitoring template instead of HardDisk either. It looks as though you want to apply the IOStat monitoring template both to HardDisk and PhysicalVolume. I don't think this is a good idea as it could easily result in collecting the exact same data twice: once for the HardDisk, and once for the corresponding PhysicalVolume. In these cases you want to override `PhysicalVolume.getGraphObjects()` to instead just show the relevant HardDisk graph(s) when looking at a PhysicalVolume.